### PR TITLE
Update build scripts to work off metabase master

### PR DESCRIPTION
**Pull request by andrewdibiasio6**
* switch git module to mainline metabase
* Add make commnds to add starburst dependecies to the mainline metabase repo
* Add logic to checkout latest metabase version to makefile
* Add metabase version to app_verisons and github action logic to extract it
* Add logic to github workflow to checkout latest version when preparing for a release.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,6 +15,12 @@ else
 	@echo "Found metabase, skipping initialization."
 endif
 
+checkout_latest_metabase_tag: clone_metabase_if_missing clean
```
Comment (hovaesco) [inline Makefile:18]: Please update README with desc about new and missing recipes.

**Review by hovaesco — APPROVED** [review]
LGTM % red CI and one comment from review.

**Comment by hovaesco** [comment]
I've seen that we are using user https://github.com/web-flow for automation. In other repos https://github.com/starburstdata-automation is used and we manage it. Do we have control over web-flow user and how is managed?

**Comment by andrewdibiasio6** [comment]
> I've seen that we are using user https://github.com/web-flow for automation. In other repos https://github.com/starburstdata-automation is used and we manage it. Do we have control over web-flow user and how is managed?

https://github.com/web-flow This account is the Git committer for all web commits (merge/revert/edit/etc...) made on GitHub.com. No, I cannot make changes to web-flow user.

**Pull request merged**